SmartTutor/Service/Recommenders/CollaborativeFilteringRecommender.cs

```diff
@@ -0,0 +1,14 @@
+﻿using SmartTutor.ContentModel;
+using System;
+using System.Collections.Generic;
+
+namespace SmartTutor.Service.Recommenders
```

Comment:
We should place the Recommenders in their own Namespace instead of the generic _Service_ namespace. We will probably discard this generic name in the end to favor a more expressive name.